src/components/form/Form.tsx

```diff
@@ -0,0 +1,166 @@
+import { FC } from 'react';
+
+import {
+  Button,
+  Checkbox,
+  Group,
+  Loader,
+  Radio,
+  Select,
+  Stepper,
+  Text,
+  Textarea,
+  TextInput,
+} from '@mantine/core';
+import { MdDelete } from 'react-icons/md';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import '@mantine/core/styles.css';
+import ModalError from '../modal/ModalError';
+import { selectErrorModal, selectShowModal } from '../redux/selectors';
+import { RootState, useAppDispatch } from '../redux/store';
+
+import useFormNavigation from './useFormNavigation';
+
+import { sendData } from 'components/api/Api';
+import { UseFormType } from 'components/App';
+import { ActivStep } from 'components/constans/EnumActiveStep';
+import { Modal } from 'components/modal/Modal';
+
+interface ICreate {
+  form: UseFormType;
+}
+
+export const Create: FC<ICreate> = ({ form }) => {
+  const { nextStep, prevStep, addString, removeString } = useFormNavigation(form);
+  const dispatch = useAppDispatch();
+  const showModal = useSelector(selectShowModal);
+  const errorModal = useSelector(selectErrorModal);
+  const { loading, active } = useSelector((state: RootState) => state.data);
```

Comment:
do not use [useSelector destructuring](https://stackoverflow.com/questions/59072200/useselector-destructuring-vs-multiple-calls)